### PR TITLE
Nav and modal

### PR DESCRIPTION
**Pull request by tanveertkd**
This PR contains the nav and modal components. The live link to preview the components can be found below.

### CODE REVIEW

**Review by srishti-maurya — APPROVED** [review]
looks good

File: components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+.modal-dismiss {
+    display: flex;
+    justify-content: flex-end;
+    position: absolute;
+    top: 1%;
+    right: 1%;
+}
+
```
Comment (adidoshi) [inline components/modal/modal.css:23]: Maybe you can add cursor pointer for dismiss button. Rest seems good to me.

**Review by adidoshi — APPROVED** [review]
Looks good.

File: components/modal/modal.css (hunk 1 above)
Comment (tanveertkd) [reply components/modal/modal.css:23]: Thanks for the suggestion. I'll make the change.

**Pull request merged**